### PR TITLE
raidboss: R3N initial triggers/timeline

### PR DESCRIPTION
**Pull request by valarnin**
~~Needs testing still. Only checked the basics in raidemulator.~~

Tested in-zone, looks good.

### CODE REVIEW

File: ui/raidboss/data/07-dt/raid/r3n.ts [hunk 1]
```diff
@@ -1,14 +1,175 @@
...
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'R3N Barbarous Barrage',
+      type: 'StartsUsing',
+      netRegex: { id: '93B2', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Knockback Towers',
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/raid/r3n.ts:73]: I didn't think to test it when I was in there, this knockback can be nullified, right? It has the correct indicator for it.

File: ui/raidboss/data/07-dt/raid/r3n.ts [hunk 2]
```diff
@@ -1,14 +1,175 @@
...
+    },
+    {
+      id: 'R3N Lariat Combo East to East',
+      type: 'StartsUsing',
+      netRegex: { id: '9ADD', source: 'Brute Bomber', capture: false },
+      durationSeconds: 15,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'East, then East',
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/raid/r3n.ts:126]:
```suggestion
          en: 'East, stay East',
```
Stylistic only, disregard as you like.

File: ui/raidboss/data/07-dt/raid/r3n.ts [hunk 3]
```diff
@@ -1,14 +1,175 @@
...
+    },
+    {
+      id: 'R3N Lariat Combo West to West',
+      type: 'StartsUsing',
+      netRegex: { id: '9ADF', source: 'Brute Bomber', capture: false },
+      durationSeconds: 15,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'West to West',
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/raid/r3n.ts:150]:
```suggestion
          en: 'West, stay West',
```
Stylistic only, disregard as you like.

File: ui/raidboss/data/07-dt/raid/r3n.ts [hunk 4]
```diff
@@ -1,14 +1,175 @@
...
+    },
+    {
+      id: 'R3N Lariat Combo West to East',
+      type: 'StartsUsing',
+      netRegex: { id: '9ADE', source: 'Brute Bomber', capture: false },
+      durationSeconds: 15,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'West to East',
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/raid/r3n.ts:138]:
```suggestion
          en: 'West, then East',
```
Probably best to ensure the style is consistent with the East -> West trigger.

File: ui/raidboss/data/07-dt/raid/r3n.txt [hunk 5]
```diff
@@ -1,8 +1,84 @@
...
+# -ii 93CE 93CF 93D2 93D7 9A81 9B44 9B59 9B5B 9B58 9AE0 9AE1 968D 93CD 9B5A 9B5B
+# -it "Brute Bomber"
+
 hideall "--Reset--"
 hideall "--sync--"
 
-0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
-
 0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+17.7 "Brutal Impact x4" Ability { id: "93D6", source: "Brute Bomber" }
```
Comment (JLGarber) [inline ui/raidboss/data/07-dt/raid/r3n.txt:12]: Here and elsewhere, do you think it's worth adding a duration?

**Review by JLGarber — APPROVED** [review]
Thanks for putting that together!

File: ui/raidboss/data/07-dt/raid/r3n.ts (hunk 1 above)
Comment (valarnin) [reply ui/raidboss/data/07-dt/raid/r3n.ts:73]: Yes, it can.

File: ui/raidboss/data/07-dt/raid/r3n.txt (hunk 5 above)
Comment (valarnin) [reply ui/raidboss/data/07-dt/raid/r3n.txt:12]: I don't know if the 4 hit lasts long enough to really be worth a `duration`? I'm fine with it either way, whatever makes more sense.

**Pull request merged**